src/http/data_loader.rs

```diff
@@ -44,6 +61,12 @@ impl HttpDataLoader {
     DataLoader::with_cache(self, tokio::spawn, HashMapCache::new()).delay(Duration::from_millis(0))
   }
 
+  pub fn to_async_data_loader_options(self, delay: usize, max_size: usize) -> DataLoader<HttpDataLoader, NoCache> {
+    DataLoader::new(self, tokio::spawn)
+      .delay(Duration::from_millis(delay as u64))
+      .max_batch_size(max_size)
+  }
```

Comment:
Should be a function inside of DataLoader. Like we have `new`. Pass args to new or use setters to configure a data loader further.